CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Precise UI Changelog
 
+## 0.7.0
```

Comment:
Is there a breaking change? From the changelog it does not seem like it.

Reply:
why? I updated the minor version, not major. PR brings a new functional - gitter badge which corresponds to minor version description

> MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.

Reply:
This is only for 1.0.0 onwords; for pre v1 semver says you may treat the minor as major etc. (i.e., the first significant digit is the major, rest follows). This has been done, e.g., by React and is a quite successful system.

Further reading on pre v1 semver: https://github.com/semver/semver/issues/333

Reply:
BTW: The Gitter badge is not a functional addition; this is a README change and has no functional impact.

Reply:
Ok, didn't know that. Thanks.
Fixed